DOCUMENTATION.md

```diff
@@ -20,7 +20,7 @@ You can install the GOV.UK Notify client package using either the command line o
   <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
   <strong class="govuk-warning-text__text">
     <span class="govuk-warning-text__assistive">Warning</span>
-    Version 2.5.0 will only work if your code is asynchronous. We have fixed this issue in version 2.5.2. [Contact us](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you have any problems.
+    Version 2.5.0 will only work if your code is asynchronous. We have fixed this issue in version 2.5.2. <a href= "https://www.notifications.service.gov.uk/support/ask-question-give-feedback"> Contact us</a> if you have any problems.
```

Comment:
Oh it looks like you might have an additional space in here that you don't need, at the start of `Contact`? Not sure if html will fix itself or not with that one